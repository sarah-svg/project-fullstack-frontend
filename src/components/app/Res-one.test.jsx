import React from 'react';
import { render, cleanup } from '@testing-library/react';

import { MemoryRouter } from 'react-router';
import ResOne from './Res-One';

describe('App component', () => {
  afterEach(() => cleanup());
  it('renders res-one', async () => {
    const { asFragment } = render(<MemoryRouter>
      <ResOne/></MemoryRouter>);
    await  expect(asFragment()).toMatchSnapshot();
  });
});
