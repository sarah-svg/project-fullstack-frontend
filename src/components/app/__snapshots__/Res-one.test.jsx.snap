// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component renders res-one 1`] = `
<DocumentFragment>
  <div>
    <figure>
      <span>
         Map id: 
      </span>
      <img />
      <figcaption>
        Map burn pile size
      </figcaption>
    </figure>
  </div>
</DocumentFragment>
`;
